letter-sender/src/main/java/com/christmas/letter/sender/dto/ChristmasLetter.java

```diff
@@ -0,0 +1,29 @@
+package com.christmas.letter.sender.dto;
+
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotEmpty;
+import jakarta.validation.constraints.NotNull;
```

Comment:
Let's cleanup unused imports. You can use the pmd plugin to do this kind of code scan.